Dockerfile

```diff
@@ -35,7 +35,7 @@ RUN echo "$(python3 --version)"
 ##### Copy python packaged
 WORKDIR /dockermount
 COPY requirements/ /dockermount/requirements/
-RUN python3 -m pip install -r requirements/requirements.txt
+RUN python3 -m pip install -r requirements/requirements.txt -r requirements/requirements-testing.txt
```

Comment:
I don't think we want to put installation of requirements-testing.txt here. If we do, won't these testing libraries be installed on every image associated with this Dockerfile?

Reply:
They will be installed in every image associated with our Dockerfile, but that's also the Dockerfile that the usaspending-test container is built on.

The other alternatives would be to create a separate Dockerfile specifically for the usaspending-test container that would install these libraries or I could add to the `command` option in the docker-compose file to run `pip install` before running `pytest`, but that's not the standard way to deal with requirements in Docker.

Reply:
I think that it's a bad idea for images to have libraries they don't need. I believe it's best practice for images to be the smallest size possible. Let's look into alternative solutions.

Reply:
Here is a snipped from Docker development best practices.

> To keep your production image lean but allow for debugging, consider using the production image as the base image for the debug image. Additional testing or debugging tooling can be added on top of the production image.

Reply:
I agree about keeping images as small as possible, but I wouldn't consider our images "small" now and these packages aren't using up tons of extra space. The most reliable alternative would be to have a completely separate Dockerfile just for the usaspending-test container, but that adds another Dockerfile that needs to be maintained in the future.

I have no issue with this new Dockerfile alternative, but it will mean another file to keep track of during things like Python version updates.